### ISSUE TITLE
getenv( 'COMPOSE_PROJECT_NAME' ) is false which causes many warnings and errors

### ISSUE THREAD

**Issue by Nikschavan** [humanmade/altis-local-server#194]
**Describe the bug**
On a project `getenv( 'COMPOSE_PROJECT_NAME' )` is returned `false` for some reason.
Because of this `$_SERVER['HTTP_HOST']` gets an incorrect value here - 
https://github.com/humanmade/altis-local-server/blob/ee5347fc492bfd56134eb3b381fe96378ac200d2/inc/namespace.php#L37

This is causing all sorts of different errors in the project, eg WP-CLI by default use incorrect value for  `DOMAIN_CURRENT_SITE` -

```
bash-5.0$ wp option get home
Error: Site '.altis.dev/' not found. Verify DOMAIN_CURRENT_SITE matches an existing site or use `--url=<url>` to override.
```

When running unit tests there are different warnings which and ultimately throw a fatal error - 
```
[24-Jul-2020 11:24:25 UTC] PHP Warning:  Asset URLs should not contain relative paths. Handle: wp-polyfill-fetch, URL: http://.altis.dev/wp-includes/js/dist/vendor/wp-polyfill-fetch.min.js?ver=3.0.0 in /usr/src/app/vendor/altis/cms/inc/namespace.php on line 352
[24-Jul-2020 11:24:25 UTC] PHP Fatal error:  Uncaught Error: Call to a member function query() on null in /usr/src/app/vendor/altis/cms/inc/namespace.php:377
Stack trace:
#0 /usr/src/app/wordpress/wp-includes/class-wp-hook.php(287): Altis\CMS\real_url_path('http://.altis.d...', 'wp-polyfill-fet...')
#1 /usr/src/app/wordpress/wp-includes/plugin.php(206): WP_Hook->apply_filters('http://.altis.d...', Array)
#2 /usr/src/app/wordpress/wp-includes/script-loader.php(198): apply_filters('script_loader_s...', 'http://.altis.d...', 'wp-polyfill-fet...')
#3 /usr/src/app/wordpress/wp-includes/script-loader.php(127): wp_get_script_polyfill(Object(WP_Scripts), Array)
#4 /usr/src/app/wordpress/wp-includes/script-loader.php(525): wp_default_packages_vendor(Object(WP_Scripts))
#5 /usr/src/app/wordpress/wp-includes/class-wp-hook.php(287): wp_default_packages(Object(WP_Scripts))
#6 /usr/src/app/wordpress/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters(NULL, Array)
#7 /usr/src/app/wordpress/wp-includes/plugin.php(544): WP_Hook->do_action(Array)
#8 /usr/src/app/wordpress/wp-includes/class.wp-scripts.php(167): do_action_ref_array('wp_default_scri...', Array)
#9 /usr/src/app/wordpress/wp-includes/class.wp-scripts.php(142): WP_Scripts->init()
#10 /usr/src/app/wordpress/wp-includes/functions.wp-scripts.php(23): WP_Scripts->__construct()
#11 /usr/src/app/wordpress/wp-includes/functions.wp-scripts.php(128): wp_scripts()
#12 /usr/src/app/wordpress/wp-includes/widgets/class-wp-widget-text.php(58): wp_add_inline_script('text-widgets', 'wp.textWidgets....')
#13 /usr/src/app/wordpress/wp-includes/class-wp-widget.php(261): WP_Widget_Text->_register_one()
#14 /usr/src/app/wordpress/wp-includes/class-wp-widget-factory.php(102): WP_Widget->_register()
#15 /usr/src/app/wordpress/wp-includes/class-wp-hook.php(287): WP_Widget_Factory->_register_widgets('')
#16 /usr/src/app/wordpress/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters('', Array)
#17 /usr/src/app/wordpress/wp-includes/plugin.php(478): WP_Hook->do_action(Array)
#18 /usr/src/app/wordpress/wp-includes/widgets.php(1765): do_action('widgets_init')
#19 /usr/src/app/wordpress/wp-includes/class-wp-hook.php(287): wp_widgets_init('')
#20 /usr/src/app/wordpress/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters(NULL, Array)
#21 /usr/src/app/wordpress/wp-includes/plugin.php(478): WP_Hook->do_action(Array)
#22 /usr/src/app/wordpress/wp-settings.php(540): do_action('init')
#23 /usr/src/app/vendor/wp-phpunit/wp-phpunit/includes/install.php(29): require_once('/usr/src/app/wo...')
#24 {main}
  thrown in /usr/src/app/vendor/altis/cms/inc/namespace.php on line 377
```

Manually setting the `$_SERVER['HTTP_HOST']` to correct domain local name fixes the above warnings.

**Expected behavior**
`$_SERVER['HTTP_HOST']` is automatically set with the correct value.

**Additional context**

**Comment by roborourke** [comment]
@Nikschavan good find - that env var is not passed through to the PHP container currently. It's easy to fix at least 😄

**Comment by roborourke** [comment]
Now released in 3.0.7 and 4.0.3
